apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ReleaseHistoryController.java

```diff
@@ -35,8 +46,29 @@ public ReleaseHistoryController(final ReleaseHistoryService releaseHistoryServic
     if (permissionValidator.shouldHideConfigToCurrentUser(appId, env, namespaceName)) {
       return Collections.emptyList();
     }
-
-   return releaseHistoryService.findNamespaceReleaseHistory(appId, Env.valueOf(env), clusterName ,namespaceName, page, size);
+    List<ReleaseHistoryBO> releaseHistoryList = releaseHistoryService
```

Comment:
how about we move the enrich logic to releaseHistoryService and modify the transformReleaseHistoryDTO2BO to add the preferred user name mapping?